src/main/java/org/ieeervce/gatekeeper/dto/User/UserAuthDTO.java

```diff
@@ -0,0 +1,42 @@
+package org.ieeervce.gatekeeper.dto.User;
+import org.ieeervce.gatekeeper.dto.Role.RoleDTO;
+import org.ieeervce.gatekeeper.dto.Society.SocietyDTO;
+
+public class UserAuthDTO {
```

Comment:
Name indicates it's used for Auth, but that's not really the case.
`UserProfileDTO` or `UserDTO`?

Reply:
Makes sense,will do that